src/platform/utilities/environment/isProduction.js

```diff
@@ -1,10 +1,4 @@
 const prodEnvironments = [
-  // www.vets.gov
-  'production',
```

Comment:
As far as our deployment tooling goes, www.vets.gov is still a production environment. I don't know what the impact of this particular change is, but just wanted to call this out and make sure it was purposeful.

Reply:
Vets.gov builds from a separate entry point now (`script/vets-gov-to-va-gov/`), so any code that might work with this module won't ever build under the `production` environment. I'm not against leaving it though if we want that there for historical purposes.

Reply:
Ah ok. I figured you were on top of it, just wanted to make sure. Carry on!